commands.go

```diff
@@ -277,15 +277,6 @@ func cmdLs(c *cli.Context) {
 
 	for _, host := range hostList {
 		if !quiet {
-			tmpHost, err := store.GetActive()
```

Comment:
I'm not sure what the purpose of this code is in the happy case so let me know if there is something obvious I have missed.